src/services/anchor-service.ts

```diff
@@ -404,7 +404,7 @@ export class AnchorService {
     if (this.includeBlockInfoInAnchorProof) {
       ipfsAnchorProof = {
         blockNumber: tx.blockNumber,
-        blockTimestamp: tx.blockTimestamp,
+        blockDate: tx.blockDate,
```

Comment:
changing the fieldname and the type from a timestamp to an ISO string would change the format of the anchor proof, which could break Ceramic as it won't know how to interpret the new anchor proof format.  Can we keep this as it was?